lib/spack/spack/test/modules/common.py

```diff
@@ -62,6 +60,7 @@ def test_modules_written_with_proper_permissions(
     assert mock_package_perms & os.stat(mock_module_filename).st_mode == mock_package_perms
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Symlinks do not work on windows yet")
```

Comment:
Same comment as above.

Reply:
I updated the skip comment. This test seems to be failing because it tests for a link existing. The current implementation of islink does not return true on windows if the file was copied. The TechX updates to symlink should resolve this issue.

Reply:
Ok, let's add a todo here as well so we remember to re-enable these once those land.